### ISSUE TITLE
[Adaptive Cards - Input Form]: Talkback is not announcing the appeared Error message while activating the submit button.

### ISSUE THREAD

**Issue by vagpt** [microsoft/Teams-AdaptiveCards-Mobile#493]
### Target Platforms

Android

### SDK Version

1.0.1

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS Version: Android 16
Device: Samsung S24
Screen Reader: Talkback
Adaptive Card Android App version: 1.0.1

**Repro Steps:**
1. Launch the Adaptive cards app on Android.
2. Turn on Talkback.
3. Navigate to 'Import' control and select it.
4. Navigate to 'Sample cards' and select it.
5. Navigate to available dropdown under 'Sample cards'.
6. Navigate to available dropdown under 'Sample cards' and select it.
7. Navigate all the controls under dropdown and select 'Input Form' card.
8. Navigate to the screen using left/right swipe gesture and observe the issue.

**Actual Result:**
While navigating the 'Input Form' screen using swipe gesture, Talkback is not announcing the appeared error message while left the field blank.

**Expected Result:**
Screen Reader should announce the appeared error message when user activates the submit button while there is no data.

**User Impact:**
It will impact the screen reader user as When TalkBack doesn’t announce the error message due to user will assume as that fields are not mandatory.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section  

### Screenshots

https://github.com/user-attachments/assets/b237cf53-cf0b-47d5-8a2f-e97b2ee1d140

### Card JSON

```JSON
NA
```
